src/main/java/com/marklogic/kafka/connect/source/MarkLogicSourceConfig.java

```diff
@@ -29,6 +29,9 @@ public class MarkLogicSourceConfig extends MarkLogicConfig {
     enum OUTPUT_TYPE {JSON, XML, CSV}
     private static final CustomRecommenderAndValidator OUTPUT_FORMAT_RV =
         new CustomRecommenderAndValidator(Arrays.stream(MarkLogicSourceConfig.OUTPUT_TYPE.values()).map(Enum::toString).toArray(String[]::new));
+    public static final String KEY_STRATEGY = "ml.source.key.strategy";
+    private static final CustomRecommenderAndValidator KEY_STRATEGY_RV =
+        new CustomRecommenderAndValidator("UUID", "TIMESTAMP", "NULL");
```

Comment:
I don't think we want "NULL" as a value - we just want this to be optional, and so I believe we want the default value to be `ConfigDef.NO_DEFAULT_VALUE`.